plugins/steve/resource-class.ts

```diff
@@ -155,24 +176,24 @@ export const STATES = {
   waitcheckin:        { color: 'warning', icon: 'tag' },
   waiting:            { color: 'info', icon: 'tag' },
   warning:            { color: 'warning', icon: 'error' },
-};
+} as StateList;
 
-export function getStatesByType(type = 'info') {
+export function getStatesByType() {
```

Comment:
Return types for functions should be stated 
```
export function getStatesByType(): StateInfoForTypes {
```
This just makes it super clear that this fn should return a specific type and makes tracking down type errors easier.

Reply:
updated